lib/rules/one-var.js

```diff
@@ -53,6 +84,34 @@ module.exports = function(context) {
         }
     }
 
+    /**
+     * Determines if there is more than one let statement in the current scope.
+     * @returns {boolean} Returns true if it is the first let declaration, false if not.
```

Comment:
Yes. This logic was just stolen from the folks before me. It's sorta poorly named, but it matches the name `hasOnlyOneVar` function that was here before I started mucking with things.